### DISCUSSION TITLE
mail regarding extention update evailable sent every hour

### DISCUSSION THREAD

**Discussion by johanpeters** [akeeba/panopticon#254] [Q&A (Community Support)] [answered]
issue #237 is not fixed regarding the amount of messages.
i still get every hour 1 message for every extention update available.
currently i have 2 extentions (1 site) that has a update available, resulting in every hour 2 mail messages regarding those update's.

**Comment by johanpeters** [comment]
checked.. i have another site with 2 updates available but the are not reported by mail (settings are correct).
i receive only messages from one site (based on order of sites its the first site with updates available).

so 1 site does sent mail (every our) and one site does not sent mails.

**Comment by johanpeters** [comment]
update.. the other site, after saved settings is now also sending mails.
so, only the amount of messages.. its keep sending me for every site and every update every hour mails regarding those updates.
and not only once .. based on your feedback i expected to receive only 1 message for every extention update, correct?

**Comment by nikosdion** [comment, marked as answer]
Fixed with https://github.com/akeeba/panopticon/issues/258
